int32 ecid
---
int32[10] flow_table
int32 size